components/automate-ui/package.json

```diff
@@ -55,6 +55,7 @@
     "rxjs": "^6.2.2",
     "sniffr": "^1.1.4",
     "web-animations-js": "^2.3.1",
+    "webdriver-manager": "^12.1.6",
```

Comment:
Important note: whenever you are committing a package change to package.json, should also commit the changes to package-lock.json.